src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(BallColors.values().length);
+        return BallColors.values()[index].toString();
```

Comment:
Use the 'name()' method instead of 'toString()' to get the String representation of enum constants, to ensure you get the exact name of the enum constant as declared.